Assets/MRTK/Services/BoundarySystem/XR2018/MixedRealityBoundarySystem.cs

```diff
@@ -32,9 +32,9 @@ public class MixedRealityBoundarySystem : BaseBoundarySystem
         /// <inheritdoc/>
         public override void Initialize()
         {
-            if (!Application.isPlaying || !XRDevice.isPresent) { return; }
-
             base.Initialize();
+
+            if (!Application.isPlaying || !XRDevice.isPresent) { return; }
```

Comment:
Can you explain a bit why we swapped the order here?

Reply:
@davidkline-ms

Reply:
I will address in a separate pr

Reply:
initialization needs to happen right away, otherwise profiles do not get read correctly.

Reply:
going to open a separate PR for the boundary system issues